_layouts/product.html

```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
         <span class="usa-tag bg-accent-warm-light text-base-darker display-inline-block">{{ item }}</span> 
         {% endfor %}
     </p>
-
+   
     <div class="grid-container padding-x-0">
         <div class="grid-row">
           <div class="tablet:grid-col tablet:padding-right-1">
             <img src="{{ page.feature-image }}" alt="{{ page.feature-image-description }}">
             <button class="usa-button usa-button--unstyled margin-y-2">View more images</button>
+            <div class="margin-y-2">
+              <button class="usa-button  usa-button--hover">Add to List</button>
```

Comment:
1. Remove usa-button--hover class. This is just for presenting documentation of UI. The hover state will occur naturally as people interact with it on a live page.
2. Let's stick with sentence case for button copy, so "Add to list".